### PR TITLE
reslove #1176: add verified space

### PR DESCRIPTION
**Pull request by Qsnh**
Feature #1176 .

Changes proposed in this pull request:
- add verified space 

image

### CODE REVIEW

File: src/components/Block/Space.vue [hunk 1]
```diff
@@ -60,7 +60,16 @@ watchEffect(() => {
...
+          <template v-if="isVerified === 1">
+            <span
+              v-tippy="{
+                content: 'Verfied space',
+                placement: 'right'
+              }"
+            >
+              <Icon name="check" size="20" />
+            </span>
+          </template>
```
Comment (Qsnh) [inline src/components/Block/Space.vue:72]: I only changed this piece of code, the rest of the code was changed by the execution of yarn lint.

File: src/locales/es-ES.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
   "strategiesPage": "Estrategias",
   "space": "Espacio",
   "spaces": "Espacios",
+  "verifiedSpace": "Verified space",
```
Comment (samuveth) [inline src/locales/es-ES.json:22]: Can you remove this from all translation files, we only need it in `default.json`

File: src/components/Block/Results.vue [hunk 3]
```diff
@@ -26,7 +26,7 @@ const choices = computed(() =>
     )
 );
 
-const getPercentage = (n, max) => 100 / max * n / 1e2;
+const getPercentage = (n, max) => ((100 / max) * n) / 1e2;
```
Comment (samuveth) [inline src/components/Block/Results.vue:29]: What is the need for this change?

File: src/components/Block/Results.vue [hunk 4]
```diff
@@ -37,7 +37,9 @@ const hideAbstain = props.space?.voting?.hideAbstain ?? false;
     :title="ts >= proposal.end ? $t('results') : $t('currentResults')"
   >
     <div v-for="choice in choices" :key="choice.i">
-      <template v-if="!(proposal.type === 'basic' && hideAbstain && choice.i === 2)">
+      <template
```
Comment (samuveth) [inline src/components/Block/Results.vue:40]: You shouldn't be changing linting so much. Please use the linting standards that we have set in the project.

**Comment by samuveth** [comment]
The arrow of the tooltip should be align to the center of the icon.
Screen Shot 2021-12-09 at 9 53 25 PM

**Comment by Qsnh** [comment]
> 

fixed.

**Review by samuveth — APPROVED** [review]
Looks good!

**Pull request merged**